relayer/chains/cosmos/types.go

```diff
@@ -67,6 +76,27 @@ type clientInfo struct {
 	header          []byte
 }
 
+type latestClientState map[string]clientInfo
```

Comment:
It's nice to have a comment noting what the key represents -- client ID? chain ID with client ID?

Reply:
done